.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  push:
+    branches: "master"
+    tags: ["*"]
+  pull_request:
+  release:
```

```suggestion
on:
  push:
    branches: ["master"]
    tags: ["*"]
  pull_request:
    branches: ["master"]
```
IIRC releases automatically create tags, so there's no need to have _both_ in the CI.